src/main/java/com/textmessenger/service/UserServiceImpl.java

```diff
@@ -30,13 +33,19 @@
   private final TemporaryTokenRepository temporaryTokenRepository;
   private UserToFrontShort userToFront;
   private final EmailService emailService;
+  private final NotificationRepository notificationRepository;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "notificationRepository" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=177&issues=AWXKdPiBlgbYnohY2kyi&open=AWXKdPiBlgbYnohY2kyi)

[ISSUE_KEY]: AWXKdPiBlgbYnohY2kyi